artworks/aitagger.go

```diff
@@ -0,0 +1,37 @@
+package artworks
+
+import (
+	"strings"
+)
+
+type AITagger struct{}
+
+func (ai AITagger) AITag(tags []string) bool {
```

Comment:
Change `tags []string` to `contents ...string`

This way a singular string can be passed to it directly without wrapping it into a slice and slices can be passed to it like this: `tags...`